src/components/common/layout/WrapperClaim.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+import React from 'react'
+import { Image, View } from 'react-native'
+import { isMobile, isMobileOnly } from 'mobile-device-detect'
+import { withStyles } from '../../../lib/styles'
+import { getScreenHeight } from '../../../lib/utils/Orientation'
+import wrapperClaimBackgroundImage from '../../../assets/wrapperClaim.svg'
+import SimpleStore from '../../../lib/undux/SimpleStore'
+Image.prefetch(wrapperClaimBackgroundImage)
+
+const wrapperClaim = props => {
+  const simpleStore = SimpleStore.useStore()
+  const shouldGrow = simpleStore.get && !simpleStore.get('isMobileSafariKeyboardShown')
+
+  const growStyle = { flexGrow: shouldGrow ? 1 : 0 }
+
+  const { backgroundColor, children, style, styles, ...rest } = props
+
+  const sizeFactor = isMobile ? 2.7 : 1.9
+  const backgroundStyle = backgroundColor
+    ? { backgroundColor: backgroundColor }
+    : {
```

Comment:
1.  const screenHeight = getScreenHeight(); then reuse screenHeight
2. this case could be pre-calculated outside component
3. you could also wrap  backgroundStyle to useMemo (to provide the same object instance until backgroundStyle becomes changed )